### PR TITLE
Bug 1377933 - Remove trailing whitespace from all perl files

### PR DESCRIPTION
**Pull request by dylanwh**

### CODE REVIEW

File: Bugzilla/DB/Pg.pm [hunk 1]
```diff
@@ -118,7 +118,7 @@ sub sql_not_regexp {
 
     $self->bz_check_regexp($real_pattern) if !$nocheck;
 
-    return "${expr}::text !~* $pattern" 
+    return "${expr}::text !~* $pattern"
```
Comment (floatingatoll) [inline Bugzilla/DB/Pg.pm]: The semicolon-free return pattern here might be worth a second pass in a different PR.

**Comment by umohm1** [comment]
I don't see any conflicting changes.
